.github/workflows/npm-diff.yml

```diff
@@ -0,0 +1,22 @@
+name: NPM lockfile changes
+on: [pull_request]
+
+jobs:
+  lockfile_changes:
+    runs-on: ubuntu-latest
+    # Permission overwrite is required for Dependabot PRs, see "Common issues" below.
+    permissions:
+      pull-requests: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: NPM lockfile changes
+        uses: emilia-capital/npm-lockfile-changes@main
```

Comment:
Getting an error that this repo doesn't exist.
Is it a repo we should create, or change this line to something else?